### PR TITLE
GH-93429: Document LOAD_METHOD removal

### LINKED ISSUE #93429: Merge `LOAD_METHOD` back into `LOAD_ATTR` for better specialization
`LOAD_METHOD` split off from `LOAD_ATTR` in 3.7 as a compiler based optimisation.

I proposed to merge it back to enable better specialization. The actual `LOAD_METHOD` instruction is left around as a virtual instruction at the moment.

See https://github.com/faster-cpython/ideas/discussions/400#discussioncomment-2867659

### PR DESCRIPTION
**Pull request by Fidget-Spinner**
Fixes #93429.

### CODE REVIEW

**Comment by markshannon** [comment]
:+1:

File: Doc/library/dis.rst [hunk 1]
```diff
@@ -887,7 +887,20 @@ iterations of the loop.
...
+
+   If the low bit of ``namei`` is set, this will attempt to load a method named
+   ``co_names[namei>>1]`` from the TOS object. TOS is popped.
+   This bytecode distinguishes two cases: if TOS has a method with the correct
+   name, the bytecode pushes the unbound method and TOS. TOS will be used as
+   the first argument (``self``) by :opcode:`CALL` when calling the
+   unbound method. Otherwise, ``NULL`` and the object return by the attribute
+   lookup are pushed.
+
+   .. versionchanged:: 3.11
```
Comment (ngnpope) [inline Doc/library/dis.rst:901]: Should this be 3.12?

```suggestion
   .. versionchanged:: 3.12
```

Comment (Fidget-Spinner) [reply Doc/library/dis.rst:901]: Yeah you're right. Thank you.

Comment (Fidget-Spinner) [reply Doc/library/dis.rst:901]: Fix in GH-93816.

**Pull request merged**